### ISSUE TITLE
Doesn't work in fresh app

### ISSUE THREAD

**Issue by samselikoff** [embermap/ember-cli-fastboot-testing#21]
```
ember new my-app
ember install ember-cli-fastboot ember-cli-fastboot-testing
ember s
> Error: Could not find module `fetch/setup` imported from `my-app/instance-initializers/setup-fetch`
```

I triggered a rebuild from Travis but it passed, I wonder why our tests aren't catching this?

**Comment by ryanto** [comment]
Related: https://github.com/ember-cli/ember-fetch/issues/166

**Comment by ryanto** [comment]
This is all fixed! Published as `0.0.9`

**Comment by samselikoff** [comment]
Nice work bro 👊
